src/cob_app/upload.clj

```diff
@@ -0,0 +1,29 @@
+(ns cob-app.upload
+  (:require [cob-app.core :refer [route DIR]]))
+
+(defn- disallow []
+  ["HTTP/1.1 405 Method Not Allowed\r\n\r\n"])
+
+(defn- write-file [input-stream output-file length]
+  (let [contents (byte-array length)]
+    (.read input-stream contents)
+    (clojure.java.io/copy contents output-file)
+    ["HTTP/1.1 200 OK\r\n\r\n"]))
```

Comment:
There are 3 ideas here in this little string, 2 of which are going to be the same across all requests:
- `HTTP/1.1 ` - each response begins with this
- `\r\n\r\n` - each response ends with this
- `200 OK` - this varies from response to response, but you'd never see something like `200 Method Not Allowed` or `405 OK` - the number & the message are always together

It seems like it'd be nice if at this level of the application, you didn't have to worry about details that are the same across all HTTP server applications. This could be an opportunity to extract some things back up to the http server (non-cob_spec-specific) level.